migrator/main.go

```diff
@@ -152,29 +148,27 @@ func upgrade(conf *config.Config, dbClone string, processBoth bool) error {
 
 	var gormDB *gorm.DB
 	var pgPool postgres.DB
-	if env.PostgresDatastoreEnabled.BooleanSetting() {
-		pgPool, gormDB, err = postgreshelper.Load(dbClone)
-		if err != nil {
-			return errors.Wrap(err, "failed to connect to postgres DB")
-		}
-		// Close when needed
-		defer postgreshelper.Close()
+	pgPool, gormDB, err = postgreshelper.Load(dbClone)
+	if err != nil {
+		return errors.Wrap(err, "failed to connect to postgres DB")
+	}
+	// Close when needed
+	defer postgreshelper.Close()
 
-		ctx := sac.WithAllAccess(context.Background())
-		ver, err := migVer.ReadVersionGormDB(ctx, gormDB)
-		if err != nil {
-			return errors.Wrap(err, "failed to get version from the database")
-		}
+	ctx := sac.WithAllAccess(context.Background())
+	ver, err := migVer.ReadVersionGormDB(ctx, gormDB)
+	if err != nil {
+		return errors.Wrap(err, "failed to get version from the database")
+	}
 
-		// If Postgres has no version and we have no bolt then we have no populated databases at all and thus don't
-		// need to migrate
-		if ver.SeqNum == 0 && ver.MainVersion == "0" && (!processBoth || boltDB == nil) {
-			log.WriteToStderr("Fresh install of the database. There is no data to migrate...")
-			pkgSchema.ApplyAllSchemas(context.Background(), gormDB)
-			return nil
-		}
-		log.WriteToStderrf("version for %q is %v", dbClone, ver)
+	// If Postgres has no version and we have no bolt then we have no populated databases at all and thus don't
+	// need to migrate
+	if ver.SeqNum == 0 && ver.MainVersion == "0" && (!processBoth || boltDB == nil) {
+		log.WriteToStderr("Fresh install of the database. There is no data to migrate...")
+		pkgSchema.ApplyAllSchemas(context.Background(), gormDB)
+		return nil
 	}
+	log.WriteToStderrf("version for %q is %v", dbClone, ver)
 
 	if boltDB == nil && !env.PostgresDatastoreEnabled.BooleanSetting() {
```

Comment:
I think this one can just go away as it checks to see if we have a legacy DB and in Rocks mode.  Since we can't be in Rocks mode any more we don't need it.